miden-crypto/src/merkle/smt/partial.rs

```diff
@@ -236,6 +293,8 @@ impl Default for PartialSmt {
 #[cfg(test)]
 mod tests {
 
+    use std::collections::{BTreeMap, BTreeSet};
```

Comment:
```suggestion
    use alloc::collections::{BTreeMap, BTreeSet};
```

That fixes the no-std test compilation.